keyboards/tada68/keymaps/mattgemmell/keymap.c

```diff
@@ -0,0 +1,50 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
+
+#define _______ KC_TRNS
```

Comment:
QMK defines this keycode alias automatically. Line 6 can be deleted.

https://github.com/qmk/qmk_firmware/blob/7c63c0a95075f6fb4dd94d020ceb5816e6bcca2c/quantum/quantum_keycodes.h#L29-L31

Reply:
Resolved in 6a75419